docker/caffe2/jenkins/common/install_rocm.sh

```diff
@@ -44,6 +44,12 @@ install_ubuntu() {
     sed -i 's/find_dependency(hip)/find_dependency(HIP)/g' /opt/rocm/rocfft/lib/cmake/rocfft/rocfft-config.cmake
     sed -i 's/find_dependency(hip)/find_dependency(HIP)/g' /opt/rocm/miopen/lib/cmake/miopen/miopen-config.cmake
     sed -i 's/find_dependency(hip)/find_dependency(HIP)/g' /opt/rocm/rocblas/lib/cmake/rocblas/rocblas-config.cmake
+
+    # for now correct the host compiler definition for forceinline here, will be part of ROCm 2.3 release
```

Comment:
I built new docker images bddppq/pytorch-rocm:253-ubuntu and bddppq/pytorch-rocm:253-centos earlier today with the vanilla rocm 2.2 and looks to me they are working pytorch fine (both locally and in our CI). What are the error messages you have seen?

Reply:
Multiple defintions of rocRAND/hipRAND functions at linking if the base compiler is clang.

Reply:
In bddppq/pytorch-rocm:253-ubuntu the base compiler is clang-7. Here is the test run on CI with the officially released rocm 2.2:

Reply:
interesting, seems to work here. well, that's great. So can we update the CI to 2.2?

Reply:
@iotamudelta Do we need to skip the two tests listed in this PR?

Reply:
@bddppq they do fail for me on the configurations listed in the commits (dual gfx906 with either Ubuntu or CentOS), so I think that makes sense.

Reply:
Ok could you revert the docker file change in this PR and then I can land the skip tests